model/BrainPadIO.h

```diff
@@ -88,6 +88,8 @@ namespace codal
             _mbed::Pin        A4;
             _mbed::Pin        A5;
             _mbed::Pin        led;
+			_mbed::Pin        scl;
```

Comment:
Align spacing

Reply:
Not sure why it's appearing like this in my code it look normal. I deleted the line again and retyped, just in case there might be some trailing spaces somewhere.